src/inline-assembly.md

```diff
@@ -384,8 +390,11 @@ assert_eq!(y, 1);
 # }
 ```
 
+r[asm.operand-type.naked_asm-restriction]
+Because `naked_asm!` defines a whole function body, it can only use `sym` and `const` operands.
```

Comment:
```suggestion
Because `naked_asm!` defines a whole function body and the compiler cannot provide any additional code to handle operands, it can only use `sym` and `const` operands.
```